packages/react/src/components/Tabs/tabPanel.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { TabPanel } from './';
+import axe from '../../axe';
+
+test('should render tab panel with two paragraphs correctly', () => {
+  render(
+    <TabPanel>
+      <p>a simple paragraph</p>
+      <p>a complicated paragraph</p>
+    </TabPanel>
+  );
+  expect(screen.getByRole('tabpanel')).toBeInTheDocument();
+});
+
+test('should pass classNames through', () => {
+  render(<TabPanel className="find--me" />);
+  expect(screen.getByRole('tabpanel')).toHaveClass('TabPanel', 'find--me');
+});
+
+test('should pass id through', () => {
+  render(<TabPanel id="I am a panelId" />);
+  expect(screen.getByRole('tabpanel')).toHaveAttribute('id', 'I am a panelId');
+});
+
+test('should support ref', async () => {
+  const ref = React.createRef<HTMLDivElement>();
+  const TabPanelwithRef = () => {
+    return <TabPanel ref={ref}>Content</TabPanel>;
+  };
+
+  render(<TabPanelwithRef />);
+  await waitFor(() => {
+    expect(ref.current).toBeInTheDocument();
+    expect(ref.current).toHaveTextContent('Content');
+  });
```

Comment:
Just a very very minor nit, I think we can just render the tab panel with the ref directly.

```tsx
render(<TabPanel ref={ref}>Content</TabPanel>)
```

And I think based on the above discussion we can also omit the `waitFor`.